paper.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Intracranial Electrode Location and Analysis in MNE-Python'
+tags:
+  - Python
+  - BIDS
+  - MNE
+  - neuroimaging
+  - iEEG
+  - sEEG
+  - ECoG
+authors:
+  - name: Alexander P. Rockhill
+    orcid: 0000-0003-3868-7453
+    affiliation: 1
+  - name: Eric Larson
+    orcid: 0000-0003-4782-5360
+    affiliation: 2
+  - name: Brittany Stedelin
+    orcid: 0000-0001-5368-4299
+    affiliation: 3
+  - name: Alessandra Mantovani
+    affiliation: 3
+  - name: Ahmed M. Raslan
+    orcid: 0000-0002-8421-7105
+    affiliation: 3
+  - name: Alexandre Gramfort
+    orcid: 0000-0001-9791-4404
+    affiliation: 4
+  - name: Nicole C. Swann
+    orcid: 0000-0003-2463-5134
+    affiliation: 1
+
+affiliations:
+ - name: Department of Human Physiology, University of Oregon, Eugene OR, USA
+   index: 1
+ - name: Institute for Learning and Brain Sciences, University of Washington, Seattle, WA, USA
+   index: 2
+ - name: Department of Neurological Surgery, Oregon Health & Science University, Portland, Oregon.
+   index: 3
+ - name: Université Paris-Saclay, Inria, CEA, Palaiseau, France
+   index: 4
+
+date: \today
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Intracranial electrophysiology analysis requires precise location and anatomical labeling of electrode recording contacts before signal processing techniques can be applied to the data. Signal processing techniques are common to other electrophysiology modalities such as magnetoencephalography (MEG) and electroencephalography (EEG) so ideally locating and labelling intracranial electrodes would be an integrated part of a single analysis software. This functional module necessary for intracranial electrophysiology analysis was added to MNE-Python [@GramfortEtAl2013a:2013] so that the suite of signal processing and analysis tools in this package can be used more easily by intracranial electrophysiology researchers.
+
+For brain electrophysiology research using stereoelectroencephalography (sEEG) and electrocorticography (ECoG), the raw electrophysiology, neurosurgical planning and neuroimaging needs to be integrated before meaningful signal processing analysis can occur. Typically, for a patient with an intracranial electrophysiology recording, a pre-operative magnetic resonance (MR) image is collected which provides detailed individual brain anatomy, a post-operative computed tomography (CT) image is collected which provides high-resolution contact locations (this can also be done with MR imaging) and the surgical plans link the names of the contacts in the recording with their positions. First, the CT must be aligned to the MR in order to associate the positions of the electrode contacts with the brain areas that they were recording from. In MNE-Python, these are aligned using a mutual information histogram-matching approach using in Dipy [@GaryfallidisEtAl2014:2014] that generally works without pre-alignment.
```

Comment:
1) again, I am not sure that the pipeline always has to be localization and then signal processing? Can  we just say before meaningful conclusions can be drawn?
2) Before "First, the CT must be aligned.." add "To implement this, first the CT..."
3) typo: "using in Dipy", change to "using Dipy" or "with Dipy"